modules/cudacodec/test/test_video.cpp

```diff
@@ -70,6 +70,10 @@ PARAM_TEST_CASE(CheckDecodeSurfaces, cv::cuda::DeviceInfo, std::string)
 {
 };
 
+PARAM_TEST_CASE(CheckInitParams, cv::cuda::DeviceInfo, std::string, bool, bool, bool)
+{
+};
```

Comment:
The test is incomplete.

Reply:
The test is implemented on line 355. As mentioned in the description it currently only checks that the new parameters have been set. To test the functionality an rstp streamer would be required.